Release 89.0.0 (#2004)
MetaMask/core: packages/queued-request-controller/CHANGELOG.md

## [Unreleased]

## [0.1.3]
### Fixed
- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))

### Changed

> - move json-rpc-engine from merged-packages to packages ([#1895](https://github.com/MetaMask/core/pull/1895))

Reviewer: Looks like the appropriate changelog for this is:
```
- Bump dependency on @metamask/json-rpc-engine to ^7.2.0
```

Reviewer: just looking at the PR description for that PR

## [Unreleased]

## [0.1.3]
### Changed
- Bump dependency on @metamask/json-rpc-engine to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))
- Bump @metamask/utils from 8.1.0 to 8.2.0 ([#1957](https://github.com/MetaMask/core/pull/1957))

### Fixed

> - Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))

Reviewer 2: What about nicknames got fixed? Is it that they're now available, and if so, available how? Also, it's not just nickname but theoretically any other network configuration information yes?

> - Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))

Author: ```suggestion
- Fix issue in extension with useRequestQueue toggled on, when a dapps selected network is different than the globally selected network, and when the dapps selected network is not a built in network, nickname would not appear in the 'toNetworkConfiguration' parameter passed to `addApproval` ([#2000](https://github.com/MetaMask/core/pull/2000))
```

Hows that?

Author: ```suggestion
- Fixes an issue in the extension when 'useRequestQueue' is enabled. The problem occurred when a DApp's selected network differed from the globally selected network, and when the DApp's chosen network was not a built-in network. Under these conditions, the nickname would not be displayed in the 'toNetworkConfiguration' parameter passed to the `addApproval` function ([#2000](https://github.com/MetaMask/core/pull/2000)).
```
...Or made 'better' by gpt

Reviewer 2: I like how it put the period after the parentheses lol.

But yes! That's very understandable!

Reviewer: Either looks good to me!

> - move json-rpc-engine from merged-packages to packages ([#1895](https://github.com/MetaMask/core/pull/1895))

Author: ```suggestion
- Bump dependency on @metamask/json-rpc-engine to ^7.2.0
```

> - Bump dependency on @metamask/json-rpc-engine to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))([#1895](https://github.com/MetaMask/core/pull/1895))

Reviewer 3: ```suggestion
- Bump dependency on @metamask/json-rpc-engine to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))
```

Author: how the heck did that happen

Author: nice1

> - Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))

Reviewer 2: Is that it did show it before and does not now, or the opposite?

> - Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))

Reviewer: LGTM!

> - Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))

Author: @mcmire bueno?

> - Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))

Author: I should reword the last part a little Ithink...

> - Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))

Author: ```suggestion
- Rectifies an issue in the extension when 'useRequestQueue' is activated. Previously, when invoking 'wallet_addEthereumChain', if the DApp's selected network was different from the globally selected network, the user was incorrectly prompted to switch the Ethereum chain prior to the 'addEthereumChain' request. With this update, 'addEthereumChain' will still be queued (due to its confirmation requirement), but the unnecessary chain switch prompt has been eliminated ([#2000](https://github.com/MetaMask/core/pull/2000)).
```
I think thats better

## [Unreleased]

## [0.1.3]
### Fixed

> - Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))

Reviewer: also want to add a comment about addEthereumChain fix?

Author: yeah good thinking
```suggestion
- Fix network nicknames ([#2000](https://github.com/MetaMask/core/pull/2000))
- Fix addEthereumChain showing switch ethereum chain dialog ([#2000](https://github.com/MetaMask/core/pull/2000))
```

### Changed

> - Change `test` build scripts to only show output for failed tests by default ([#1949](https://github.com/MetaMask/core/pull/1949))

Reviewer: I don't think we need to include this since consumers aren't impacted

> - Change `test` build scripts to only show output for failed tests by default ([#1949](https://github.com/MetaMask/core/pull/1949))

Author: ```suggestion
```

- move json-rpc-engine from merged-packages to packages ([#1895](https://github.com/MetaMask/core/pull/1895))
- Bump @metamask/utils from 8.1.0 to 8.2.0 ([#1957](https://github.com/MetaMask/core/pull/1957))

> - Bump @metamask/auto-changelog from 3.4.2 to 3.4.3 ([#1997](https://github.com/MetaMask/core/pull/1997))

Reviewer: I don't think we need to include this since it's a devdep

> - Bump @metamask/auto-changelog from 3.4.2 to 3.4.3 ([#1997](https://github.com/MetaMask/core/pull/1997))

Author: ```suggestion
```
